build.gradle

```diff
@@ -13,14 +13,18 @@ buildscript {
 
 //add dependency repositories
 repositories {
-	maven {
-		name = "CB Repo"
-		url = "http://chickenbones.net/maven"
-	}
-	maven {
-		name = "JEI Maven"
-		url = "http://dvs1.progwml6.com/files/maven"
-	}
+    maven {
+        name = "CB Repo"
+        url = "http://chickenbones.net/maven"
+    }
+    maven {
+        name = "JEI Maven"
+        url = "http://dvs1.progwml6.com/files/maven"
+    }
+    maven {
+        name = "CurseForge"
+        url = "https://minecraft.curseforge.com/api/maven/"
+    }
```

Comment:
Do you add new run configurations for ForgeGradle tasks other than the `RunClient`/`RunServer` ones provided? Also, that's actually terrifying that you had to create an entire application to setup and modify a workspace... Did that set off some alarm bells for you or was it just me? I just feel like Gradle has a much easier way to be able to modify many mods at the same time. Just using composite builds would allow you to modify any of the dependent mods at will. I'm not sure how you would be able to test with a mod enabled or disabled quickly, but I don't think your application does that currently. Being able to depend on a Forge jar is a shortcoming of the ForgeGradle plugin, but there may be a way around it. I just think you should really reevaluate using Gradle instead of a custom-made program since Intellij already can dynamically import/translate a complex Gradle project structure into a module hierarchy similar to the one shown there. I think composite builds bridges the gap that your multi-project/directory-thing importing program does, with the bonus that it doesn't need to be maintained since Gradle itself supports it.